cypress/integration/pageobjects/graphical-query/graphical-query-page.ts

```diff
@@ -0,0 +1,40 @@
+import { getIframeBody } from '../../../support/commands';
+
+export default class GraphicalQueryPage {
+
+    selectStudyName (studyName: string) {
+        getMainIframeDocumentWaitToLoad().xpath(`//div[@id='trials']//input`).should('be.visible').type(studyName, { force: true, delay: 0 });
+        getIframeBody().xpath(`//span[contains(@class,'select2-results')]//ul[@class='select2-results__options']/li[text()='${studyName}']`).should('be.visible').click();
+    }
+
+    selectObservationLevel () {
+        getIframeBody().find('[data-test="observationLevelSelect"]').should('exist').select('PLOT');
+    }
+
+    clickLoadQuery () {
+        getIframeBody().find('#load-brapi').should('exist').click();
+    }
+
+    addRangeOfValues () {
+        getIframeBody().find('button[data-test="addFilterButton"]').should('exist').click();
+        getIframeBody().find('a[data-test="newRangeOption"]').should('exist').click();
+        getIframeBody().find('select[data-test="selectTrait"]').should('exist').select('Aflatox_M_ppb');
```

Comment:
Kindly pass this value as an argument so that we can re-use this function in othe traits